http.slide

```diff
@@ -28,3 +28,30 @@ pawel.kowalak@gmail.com
     "temp": 18,
     "windy": false
   }
+
+* Simplest HTTP server
+
+.code http/w.go
+
+* ...with error handling
```

Comment:
Instead of coping example it's possible to show only one func from the file. Then you can have one big file and show just part of it on specified slide.

 Example:
.code go-concurrency-basics/pipeline.go /func double/,/^}/